README.md

```diff
@@ -11,32 +11,45 @@ Link to any figshare resources as follows: [![pdf manual](https://img.shields.io
 About
 -----
 
-Describe your software in one paragraph. Explain its application, what it can
-do and who should use it. If your software is one of the CIG software projects
-end with the following sentence:
-The project is supported by CIG (http://geodynamics.org).
+`Describe your software in one paragraph. Explain its application, what it can
+do and who should use it. `
+
+SOFTWARE_TEMPLATE is an example repository and serves as a guide to new and existing projects to help your project apply software best practices. This repository follows CIG software  [best practices](https://github.com/geodynamics/best_practices) and is a companion to its [contribution checklist]().  
+
+The checklist and this repository is meant as a guide to prepare your software for CIG review.
+
+The project is supported by Computational Infrastructure for Geodynamics (CIG)
+which is supported by the National Science Foundation award NSF-0949446 and NSF-1550901
+(http://geodynamics.org).
 
 
 Installation instructions
 -------------------------
 
-Either refer to the [INSTALL](INSTALL) file, or link to another place where you keep installation instructions.
+`Either refer to the`[INSTALL](INSTALL) `file or link to another place where you keep installation instructions.`
 
+This repository contains no code or executable files.
 
 Running and extending SOFTWARE_TEMPLATE
 ---------------------------------------
 
-Refer to where the user can find instructions on how to run and modify the software. We also recommend adding a quick-start guide here.
+`Refer to where the user can find instructions on how to run and modify the software. We also recommend adding a quick-start guide here.`
+
+New projects should begin by placing a copy of this repository into your project repository and modifying the contents of each file as appropriate.
 
 
 Citing SOFTWARE_TEMPLATE
 ------------------------
 
-Include information about how to cite SOFTWARE_TEMPLATE, e.g. using https://geodynamics.org/cig/abc. We also recomment adding a codemeta file, e.g. generated using https://codemeta.github.io/codemeta-generator/ to make your software citable.
+`Include information about how to cite SOFTWARE_TEMPLATE. Consider adding citations to both publications and the software itself. For examples, see:` [https://geodynamics.org](https://geodynamics.org/cig/abc).
 
+`We also recommend adding a codemeta file (`codemeta.json`). Generate this file to make your code citable:`[https://codemeta.github.io/codemeta-generator/]( https://codemeta.github.io/codemeta-generator/ ).
```

Comment:
Maybe make that a link to the file in the repo?

```suggestion
`We also recommend adding a codemeta file (`[codemeta.json](codemeta.json)`). Generate this file to make your code citable:`[https://codemeta.github.io/codemeta-generator/]( https://codemeta.github.io/codemeta-generator/ ).
```

Reply:
ok to merge. Thanks.